src/toast/ops/stokes_weights/kernels_jax.py

```diff
@@ -191,18 +133,18 @@ def stokes_weights_IQU_interval(
         quats, (quat_index, intervals, ALL)
     )  # quats[quat_index,intervals,:]
 
-    IAU_sign = 1.0 - 2.0 * IAU
-
+    # Are we using a half wave plate?
     if hwp.size == 0:
-        # Not using a half wave plate
-        new_weights_interval = stokes_weights_IQU_inner(
-            epsilon, cal, gamma, quats_interval, IAU_sign
-        )
+        # No half wave plate
+        hwp_interval = jnp.zeros_like(intervals.indices)
```

Comment:
Are we sure that this allocation will be optimized away inside the `inner` function?

Reply:
Yes, it is optimized out by the compiler.